scripts/js.js

```diff
@@ -57,18 +57,35 @@ function setCursor() {
     var scaleX = canvas.getBoundingClientRect().width / canvas.width;
     cursorCanvas.width = brushSize * 2 * scaleX;
     cursorCanvas.height = brushSize * 2 * scaleX;
+
+    // two cursors – for visibility against dark backgrounds
     var cursorCtx = cursorCanvas.getContext('2d');
+    var outlineCtx = cursorCanvas.getContext('2d');
```

Comment:
Do we need two contexts for this? I think it would work with just the one, as long as the one path is finished and drawn before we start the other.

Reply:
Good point, thanks. Working on that change now!